rca/programmes/models.py

```diff
@@ -207,6 +208,37 @@ def __str__(self):
         return self.title
 
 
+class ProgrammeStudyMode(models.Model):
+    """
+    For instance, full-time, part-time, etc.
+    """
+
+    title = models.CharField(max_length=128)
+
+    def __str__(self):
+        return self.title
```

Comment:
When we come to do the filtering, this might need a slug - I'd suggest looking at the current taxonomies used to filter programmes and matching those in terms of what the model is like

Reply:
Also I would suggest we limit the creation of these instances to 2 - seen as though we will be using it as a boolean?

Reply:
Thanks, updated in 0f1ee3fb